Inline `TextTrack::invalidateTrackIndex`

nt1m opened the pull request:
#### b1dd9328b1de545fd81ad38c036eb922aa834ed6
<pre>
Inline `TextTrack::invalidateTrackIndex`
<a href="https://bugs.webkit.org/show_bug.cgi?id=283429">https://bugs.webkit.org/show_bug.cgi?id=283429</a>
<a href="https://rdar.apple.com/140287648">rdar://140287648</a>

Reviewed by Ryosuke Niwa and Chris Dumez.

Silence some smart pointer errors with unified sources shifting around. See <a href="https://github.com/WebKit/WebKit/pull/34778">https://github.com/WebKit/WebKit/pull/34778</a>

* Source/WebCore/html/track/TextTrack.cpp:
(WebCore::TextTrack::invalidateTrackIndex): Deleted.
* Source/WebCore/html/track/TextTrack.h:
(WebCore::TextTrack::invalidateTrackIndex):

Canonical link: <a href="https://commits.webkit.org/286866@main">https://commits.webkit.org/286866@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d7ea975877225e1de07701010653afe3402e5138

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/77337 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/56372 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/30253 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/81910 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/28619 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/79454 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/65520 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/4669 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/60591 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/18626 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/80404 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/50567 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/66383 "Passed tests") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/40891 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/47969 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/23882 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/26942 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/69080 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/24220 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/83326 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/4717 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/3203 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/68859 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/4873 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/66354 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/68116 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/17010 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/12111 "Passed tests") | [⏳ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-WK2-Intel-Tests-EWS "Waiting to run tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/4664 "Built successfully") | [  ~~🛠 mac-safer-cpp~~](https://ews-build.webkit.org/#/builders/120/builds/7479 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [❌ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/4683 "Failed to checkout and rebase branch from PR 36908") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/8118 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/6442 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

cdumez commented on Source/WebCore/html/track/TextTrack.h:
```diff
@@ -109,7 +109,11 @@ class TextTrack : public TrackBase, public EventTarget, public ActiveDOMObject {
 
     int trackIndex();
-    void invalidateTrackIndex();
+    inline void invalidateTrackIndex()
```
You don't need the `inline` keyword in the method implementation is already inlined, like it is here.

nt1m committed:
Inline `TextTrack::invalidateTrackIndex`
https://bugs.webkit.org/show_bug.cgi?id=283429
rdar://140287648

Reviewed by Ryosuke Niwa and Chris Dumez.

Silence some smart pointer errors with unified sources shifting around. See https://github.com/WebKit/WebKit/pull/34778

* Source/WebCore/html/track/TextTrack.cpp:
(WebCore::TextTrack::invalidateTrackIndex): Deleted.
* Source/WebCore/html/track/TextTrack.h:
(WebCore::TextTrack::invalidateTrackIndex):

Canonical link: https://commits.webkit.org/286866@main
